.github/labeler.yml

```diff
@@ -0,0 +1,12 @@
+node:
+  - lib/msal-node/**/*
+angular:
```

Comment:
There is already an Angular label, is that okay?

Reply:
Yes. All of these already exist. If it doesn't work it should get logged in the Actions tab

Reply:
Ah right.